.github/workflows/jest.yml

```diff
@@ -0,0 +1,55 @@
+name: Jest Tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'eask'
+      - '**.yml'
+      - lisp/**
+      - cmds/**
+      - src/**
+      - test/**
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        emacs-version: [snapshot]
+        experimental: [true]
+
+    steps:
+    - uses: jcs090218/setup-emacs@master
+      with:
+        version: ${{ matrix.emacs-version }}
+
+    - uses: actions/checkout@v4
+
+    - name: Prepare Eask (Unix)
+      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      run: |
+        chmod -R 777 ./
+        .github/scripts/setup-eask
+
+    - name: Prepare Eask (Windows)
+      if: matrix.os == 'windows-latest'
+      run: .github/scripts/setup-eask.ps1
+
+    - name: Testing...
+      run: |
+        npm run test-docker
```

Comment:
I'm a bit confused with the script name `test-docker`. 🤔

Reply:
Good point, it's confusing with the test file `docker.test.js`
Perhaps one of:
- `test-ci`
- `ci`
- `test-unsafe`

Which would you prefer?

Reply:
Let's go with `test-unsafe` for now. 🤔 We can switch it to `xxx-ci` later, specifically for CI.